XamlControlsGallery/ControlPages/ListViewPage.xaml.cs

```diff
@@ -56,6 +85,242 @@ private void SelectionModeComboBox_SelectionChanged(object sender, SelectionChan
+                Contact tmp = (Contact)e.Items[0];
+
+                e.Data.SetText(tmp.FirstName + " " + tmp.LastName + " " + tmp.Company);
+                e.Data.RequestedOperation = DataPackageOperation.Move;
+            }
+        }
+
+        private void Target_DragEnter(object sender, DragEventArgs e)
+        {
+            // We don't want to show the Move icon
+            e.DragUIOverride.IsGlyphVisible = false;
+        }
+
+        //===================================================================================================================
+        // Grouped Headers Example
+        //===================================================================================================================
+        private void ToggleSwitch_Toggled(object sender, RoutedEventArgs e)
+        {
+            if (StickySwitch != null)
+            {
+                if (StickySwitch.IsOn == true)
+                {
+                    stackPanelObj.AreStickyGroupHeadersEnabled = true;
+                }
+                else
+                {
+                    stackPanelObj.AreStickyGroupHeadersEnabled = false;
+                }
+            }
+        }
+
+        private void StackPanel_loaded(object sender, RoutedEventArgs e)
+        {
+            stackPanelObj = sender as ItemsStackPanel;
+        }
+
+        //===================================================================================================================
+        // Inverted List Example
+        //===================================================================================================================
+
+        private void AddItemToEnd()
+        {
+
+            InvertedListView.Items.Add(
+                new Message("Message " + ++messageNumber, DateTime.Now.ToString(), "Right")
+                );
+        }
+
+        private void MessageRecieved(object sender, RoutedEventArgs e)
+        {
+            InvertedListView.Items.Add(
+                new Message("Message " + ++messageNumber, DateTime.Now.ToString(), "Left")
+                );
+        }
+    }
+
+    public class Message
+    {
+        public string MsgText { get; private set; }
+        public string MsgDateTime { get; private set; }
+        public string MsgAlignment { get; set; }
+        public SolidColorBrush BgColor { get; set; }
+        public Message(string text, string datetime, string align)
+        {
+            MsgText = text;
+            MsgDateTime = datetime;
+            MsgAlignment = align;
+
+            // If recieved message, use yellow background
+            if (MsgAlignment == "Left")
+            {
+                BgColor = (SolidColorBrush)Application.Current.Resources["SystemControlBackgroundAccentBrush"]; 
+            }
+
+            // If sent message, use light gray
+            else
+            {
+                BgColor = (SolidColorBrush)Application.Current.Resources["SystemControlBackgroundBaseMediumLowBrush"];
```

Comment:
I'm not sure why, but in dark theme the sent messages' background disappears.

Reply:
SystemControlBackgroundBaseMediumLowBrush doesn't seem to be rendering correctly in dark theme...it should be using #99FFFFFF color, but that's not what I'm seeing on my machine...everything just looks black.

Reply:
This is probably because the system theme is still light while the app theme I changed to dark. In markup, I would make sure they're all ThemeResource (as opposed to StaticResource). But in code-behind, I'm not sure how to force the right color.

Reply:
Not perfect, but found a different brush that worked.  Changed all the static resource brushes to theme resources as well.